snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/Emitter.java

```diff
@@ -881,6 +898,13 @@ public long getByteLimitPost() {
         return this.byteLimitPost;
     }
 
+    /**
+     * @return the customPath
+     */
+    public String getCustomPath() {
```

Comment:
This should probably be `getCustomPostPath` to better represent the variable.

Reply:
Thanks for the tip! It's done. :smile: